eng/pipelines/templates/jobs/integration.yml

```diff
@@ -11,7 +15,7 @@ jobs:
 
   - template: /eng/pipelines/templates/steps/publish-to-dev-feed.yml
```

Comment:
I just tried publishing to DevFeed from this PR commit, and it looks like there’s a version conflict, it seems to have used the release version instead of the dev version.

 https://dev.azure.com/azure-sdk/internal/_build/results?buildId=5263402&view=logs&j=a904a143-1915-56b9-4d0a-05f5515330ce&t=abbbda38-f52e-5b1e-5500-75c6b7d62760